registrations/api.py

```diff
@@ -583,7 +583,7 @@ class SignUpViewSet(
         filters.OrderingFilter,
     ]
     ordering_fields = ("first_name", "last_name")
-    ordering = ("first_name", "last_name")
+    ordering = ("id",)
```

Comment:
In previous uses of `ordering` the field value(s) have always been included in `ordering_fields` and I think that's preferable for predictable behaviour. Is the deviation intentional? 

With the current approach `ordering=id` would seemingly work as it's the default `ordering`,  `ordering=-id` wouldn't work and `ordering=id,first_name` would actually be equivalent to `ordering=first_name` since I believe `OrderingFilter` just drops tokens that are not in ordering_fields.

Reply:
Very good point. I think for now the UI does not use ordering request params to get the signups so this would work indeed but perhaps I'll add those first_name and last_name back to avoid the confusion you mentioned above.

Reply:
I changed it to have the names and added tests for sort

Reply:
hmm, shouldn't this rather be 
```python
    ordering_fields = ("id", "first_name", "last_name")
    ordering = ("id",)
```

Reply:
> hmm, shouldn't this rather be
> 
> ```python
>     ordering_fields = ("id", "first_name", "last_name")
>     ordering = ("id",)
> ```

:point_up: yeah, I meant this

Reply:
Thank you for clarification. Once again updated.